site/content/en/reference/live/preview/_index.md

```diff
@@ -47,5 +49,9 @@ DIRECTORY:
 ```
 --destroy:
   If true, dry-run deletion of all resources.
+
+--server-side:
```

Comment:
We should add `--field-manager` and `--force-conflicts` here too.

Reply:
Done.